src/main/java/kr/codesqaud/cafe/service/ArticleService.java

```diff
@@ -0,0 +1,28 @@
+package kr.codesqaud.cafe.service;
+
+import kr.codesqaud.cafe.domain.Article;
+import kr.codesqaud.cafe.repository.ArticleRepository;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+public class ArticleService {
+    private final ArticleRepository articleRepository;
+
+    public ArticleService(ArticleRepository articleRepository) {
+        this.articleRepository = articleRepository;
+    }
```

Comment:
```suggestion
@Autowired
ArticleRepository articleRepository;
```
이렇게 사용하면 코드가 더 깔끔하지 않을까요? 왜 생성자 주입을 사용하셨나요?

Reply:
생성자가 하나 있을 때는 `@Autowired`를 사용하는 것이 의미 없다는 말을 들었습니다.
그 말을 듣기 전, 이를 시도해 보았었는데 `Field injection is not recommended`라는 것을 인텔리제이가 알려주었습니다.

사실 여기까지만 알고 그냥 넘어갔었지만 지금 kyu의 말씀으로 의존성 주입에 대해 더 공부해볼 것 같습니다.
더 생각해볼 기회를 만들어주셔서 정말 감사드립니다 😄